content/blog/2018-04-23-sourcebots-2018-is-over.md

```diff
@@ -0,0 +1,22 @@
+---
+title: SourceBots 2018 Is Over!
+date: 2018-04-23
+---
+
+SourceBots 2018 is over, and sees Collyer's take the 1st place trophy, in an epic final between RGS Guildford, Barton Pevril and Brokenhurst.
```

Comment:
The "between" here makes it sound like (somehow) Collyer's weren't actually in the final.

Reply:
7b4e0262391161840101c5487eefd57f77afceac rewrites this entirely.